multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -56,6 +69,31 @@ coreos:
         [Install]
         WantedBy=multi-user.target
 
+{{ if eq .ContainerRuntime "rkt" }}
+    - name: rkt-api.service
+      command: start
+      enable: true
+      content: |
+        [Service]
+        ExecStart=/usr/bin/rkt api-service
+        Restart=always
+        RestartSec=10
+
+    - name: load-rkt-stage1.service
+      command: start
```

Comment:
In order for `RequiredBy` to work, this unit has to be `enabled:true`. Also get rid of `command: start`, the whole point of `RequiredBy` is that you don't explicitly start dependencies- in the same way you don't explicitly install apt dependencies. 

There's a really important distinction to not munge between "this unit is active as a dependency of these other units" VS "this unit is active because the operator said so (`command: start`)"